documentation/upgrading_to_bolt_3.md

```diff
@@ -0,0 +1,82 @@
+# Upgrading to Bolt 3.0
+
+It's almost time for Bolt 3.0! This page contains a list of things you can do to prepare for the
+upcoming release. For an exhaustive list of the things we're changing, see [Changes coming in Bolt
```

Comment:
```suggestion
upcoming release. We'll add to this page as we near the release and implement more of the expected changes and deprecations, so if you're making the changes ahead of time, be sure to check in every couple weeks to see if we've added anything else to the instructions below. For an exhaustive list of the things we're changing, see [Changes coming in Bolt
```

Reply:
I thought about this a bit more last night and decided we should hint at the fact that this is a living page until the release goes out. Or someone might make the changes early and miss stuff we add later.